### ISSUE TITLE
C4AI Gradient: Spec out how the AI gradient works across all components

### ISSUE THREAD

**Issue by mattrosno** [carbon-design-system/carbon#15072]
> [!WARNING]  
> **Blocked by**
> - #15071 

**Notes**

- See 10/17/23 notes in https://github.com/carbon-design-system/carbon/issues/14928
- Include requirements for gradient directionality, scalability (width and height), etc.
```[tasklist]
### Tasks
- [ ] Spec how the gradient looks next to the slug, as gradient is never used without the slug
- [ ] Directionality
- [ ] Scalability
```

**Comment by mattrosno** [comment]
@aagonzales @tw15egan would you say the 3 tasklist items in this issue are complete?

**Comment by aagonzales** [comment]
@mattrosno Mostly, TJ has done a first of everything. Design wise we are still iterating on the visual of the gradient but that's kind of an open issue as we refine things. There's been a lot of back and forth. We could maybe close this as complete though since there is an MVP of this done in code now. And we can open another issue later for design updates when needed.
